.circleci/config.yml

```diff
@@ -441,6 +441,10 @@ jobs:
             - run:
                 name: ktlint
                 command: ./gradlew ktlint
+            - run:
+                name: check release runtime Mapbox dependencies
+                command: |
+                  ./gradlew sdk:dependencyInsight --configuration releaseRuntimeClasspath --dependency com.mapbox.maps --no-daemon --no-parallel | grep -vzq 'FAILED'
```

Comment:
Wouldn't gradle fail later actually if dependency is not resolved?

Reply:
This is just to fail fast in the CI workflow